sdk/cmake/msvc.cmake

```diff
@@ -205,12 +205,16 @@ else()
     set(CMAKE_RC_COMPILE_OBJECT "<CMAKE_RC_COMPILER> /nologo <INCLUDES> <FLAGS> <DEFINES> ${I18N_DEFS} /fo <OBJECT> <SOURCE>")
 endif()
 
+if(MSVC_VERSION GREATER_EQUAL 1936)
+    set(_quiet_flag "-quiet")
```

Comment:
Why not using the slash syntax for this switch, as done for the other ones?

Reply:
Because arm seems to require dashes for all commands, so I tried that. Since it doesnt work, I'll switch to slash.